php-nginx/php.ini

```diff
@@ -586,7 +586,7 @@ html_errors = Off
 ; Example:
 ;error_log = php_errors.log
 ; Log errors to syslog (Event Log on Windows).
-error_log = /var/log/app_engine/php-error.log
+error_log = /var/log/app_engine/application.log
```

Comment:
What this change will give us? Will this give you a better name in the dropdown in the Developers Console?